hoot-core/src/main/cpp/hoot/core/io/OsmApiChangesetElement.h

```diff
@@ -287,8 +293,9 @@ class XmlRelation : public XmlElement
 };
 
 /** Custom sorting function to sort IDs from -1 to -n followed by 1 to m */
-struct osm_id_sort
+class osm_id_sort
```

Comment:
Best practice* says that an object description that contains member functions should be a `class` and not a `struct`.